README.md

```diff
@@ -0,0 +1,151 @@
+    Default output format [None]: json
+
+To use environment variables, do the following:
+
+    $ export AWS_ACCESS_KEY_ID=<access_key>
+    $ export AWS_SECRET_ACCESS_KEY=<secret_key>
+
+To use the shared credentials file, create an INI formatted file like this:
+
+    [default]
+    aws_access_key_id=MYACCESSKEY
+    aws_secret_access_key=MYSECRETKEY
+
+    [testing]
+    aws_access_key_id=MYACCESKEY
+    aws_secret_access_key=MYSECRETKEY
+
+and place it in `~/.aws/credentials` (or in `%UserProfile%\.aws/credentials` on Windows). If you wish to place the shared credentials file in a different location than the one specified above, you need to tell aws-cli where to find it. Do this by setting the appropriate environment variable:
+
+    $ export AWS_SHARED_CREDENTIALS_FILE=/path/to/shared_credentials_file
+
+To use a config file, create an INI formatted file like this:
+
+    [default]
+    aws_access_key_id=<default access key>
+    aws_secret_access_key=<default secret key>
+    # Optional, to define default region for this profile.
+    region=us-west-1
+
+    [profile testing]
+    aws_access_key_id=<testing access key>
+    aws_secret_access_key=<testing secret key>
+    region=us-west-2
+
+and place it in `~/.aws/config` (or in `%UserProfile%\.aws\config` on Windows). If you wish to place the config file in a different location than the one specified above, you need to tell aws-cli where to find it. Do this by setting the appropriate environment variable:
+
+    $ export AWS_CONFIG_FILE=/path/to/config_file
+
+As you can see, you can have multiple `profiles` defined in both the shared credentials file and the configuration file. You can then specify which profile to use by using the `--profile` option. If no profile is specified the `default` profile is used.
+
+In the config file, except for the default profile, you **must** prefix each config section of a profile group with `profile`. For example, if you have a profile named \"testing\" the section header would be `[profile testing]`.
+
+The final option for credentials is highly recommended if you are using aws-cli on an EC2 instance. [IAM Roles](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html) are a great way to have credentials installed automatically on your instance. If you are using IAM Roles, aws-cli will find and use them automatically.
+
+In addition to credentials, a number of other variables can be configured either with environment variables, configuration file entries, or both. See the [AWS Tools and SDKs Shared Configuration and Credentials Reference Guide](https://docs.aws.amazon.com/credref/latest/refdocs/overview.html) for more information.
+
+For more information about configuration options, please refer to the [AWS
+CLI Configuration Variables topic](http://docs.aws.amazon.com/cli/latest/topic/config-vars.html#cli-aws-help-config-vars). You can access this topic from the CLI as well by running `aws help config-vars`.
+
```

Comment:
I feel like it would be nice to have some small examples of using the CLI to illustrate it to people. We lost that in removing some of the sections from the rst document. Like a simple `aws --version` check, a sample`aws s3 ls` command, and a help command `aws help` along with the link to the reference docs would be helpful to get people started who are brand new.